lib/pg_ha_migrations/unsafe_statements.rb

```diff
@@ -91,15 +106,21 @@ def self.delegate_raw_method_to_migration_base_class(method_name)
   disable_or_delegate_default_method :rename_column, ":rename_column is NOT SAFE! Explicitly call :unsafe_rename_column to proceed"
   disable_or_delegate_default_method :rename_table, ":rename_table is NOT SAFE! Explicitly call :unsafe_rename_table to proceed"
 
-  def unsafe_create_table(table, options={}, &block)
+  # Note that unsafe_* methods defined below do not run dependent object checks
+
+  def unsafe_change_table(*args, &block)
+    raise PgHaMigrations::UnsafeMigrationError.new(":change_table is too generic to even allow an unsafe variant. Use a combination of safe and explicit unsafe migration methods instead")
+  end
```

Comment:
I don't think we've ever done something like this before, so open to opinions

Reply:
I think it's fine, but we should be sure it's documented.

Reply:
Done in https://github.com/braintree/pg_ha_migrations/pull/113/commits/bee6f73904f3fad69572422813c378d51a2d1140